lightning/src/ln/channel.rs

```diff
@@ -5829,6 +5848,7 @@ impl<Signer: Sign> Writeable for Channel<Signer> {
 			(17, self.announcement_sigs_state, required),
 			(19, self.latest_inbound_scid_alias, option),
 			(21, self.outbound_scid_alias, required),
+			(23, self.counterparty_forwarding_info.as_ref().map(|info| info.outbound_htlc_maximum_msat).unwrap_or(None), option),
```

Comment:
nit: You don't need the `unwrap_or(None)` - its already None if the unwrap fails :)

Reply:
Removing the `unwrap_or(None)` would result in `Some(Some(outbound_htlc_maximum_msat))` if a value for `outbound_htlc_maximum_msat` exists, as both `counterparty_forwarding_info` and `outbound_htlc_maximum_msat` are optional fields. But perhaps there is a neater way to solve that :)